src/components/layout/TheHeader.vue

```diff
@@ -11,82 +12,95 @@
   <v-app-bar
     :elevation="2"
     color="#001E49"
-    height="40"
+    height="60"
   >
     <!-- Logo and title -->
     <v-toolbar-title>
       <v-row no-gutters>
         <v-col
+          class="hidden-sm-and-down"
           cols="auto"
-          class="mr-2"
         >
           <v-img
-            alt="SchulPortalLogo"
-            :src="SchulPortalLogo"
-            :width="30"
-            :height="30"
+            alt="SchulPortalWortLogoBlau"
+            :src="SchulPortalWortLogoBlau"
+            :width="354"
+            :height="60"
           />
         </v-col>
 
         <!-- Hide this column on small screens and below -->
         <v-col
           cols="auto"
-          class="hidden-sm-and-down"
+          class="hidden-md-and-up"
         >
-          <div class="nav-title">SCHULPORTAL <span class="normal-weight">SH</span></div>
+          <v-img
```

Comment:
Warum wird immer das Bild importiert? Da ist keine dynamische URL dran. Reicht nicht auch:
```suggestion
          <img
            alt="Schulportal logo"
            class="logo"
            src="@/assets/logos/Schulportal_SH_Wort_Bildmarke_RGB_Anwendung_HG_Weiss.svg"
            width="598"
          />
```